lib/provider/data_types.go

```diff
@@ -166,3 +166,17 @@ type SnapshotParameters struct {
 	// tags for the snapshot
 	SnapshotTags SnapshotTags `json:"tags,omitempty"`
 }
+
+// UpdatePVC ...
+type UpdatePVC struct {
+	ID         string   `json:"id,omitempty"`
```

Comment:
can you remove omitempty for ID, CRN , Name and capacity

Reply:
We can take up as improvement